extra/github-actions/build-mac.yml

```diff
@@ -13,16 +13,16 @@
     curl -L https://github.com/madler/zlib/releases/download/v$ZLIB_VERSION/zlib-$ZLIB_VERSION.tar.gz | tar xz
     cd zlib-$ZLIB_VERSION
     ./configure
-    make && make install
+    sudo make && sudo make install
```

Comment:
Why does this become necessary?

Reply:
No access without sudo to usr/local/local folder to install stuff since macos13, maybe. Error was:
```
+ make install
mkdir: /usr/local/share/man/man3: Permission denied
make: [install-libs] Error 1 (ignored)
mkdir: /usr/local/lib/pkgconfig: Permission denied
make: [install-libs] Error 1 (ignored)
rm -f /usr/local/lib/libz.a
cp libz.a /usr/local/lib
cp: /usr/local/lib/libz.a: Permission denied
make: *** [install-libs] Error 1
```